inc/blocks/ab-test/components/variant-panel.js

```diff
@@ -60,6 +84,13 @@ const VariantPanel = ( { title, variant, variants, onMouseDown } ) => {
 					} );
 				} }
 			/>
+			<Preview
+				href={ `/?p=${ postId }&set_test=ab_block_${ postId }:${ index }` }
+				target="_ab_test_preview"
+			>
+				{ __( 'Preview', 'altis-analytics' ) }
+				<Icon icon="external" />
+			</Preview>
```

Comment:
The `!!` ensure that the value of `postId` is not just falsy but definitely a boolean. In some cases say if it was a `0` and we just wrote `postId && ( ... )` then it would output "0" as a string.

Reply:
This does work with a draft post.

I do agree with your suggestion on adding `preview=xyz` to the preview URL's. Especially for the need to have the url to set a default state of a tab. Example were if the user clicked preview from "Variant C" when they previewed, "Variant C" should be active.

Reply:
Cool, the set_test param can be used to set the default tab then